web/src/classic/components/organisms/Settings/ProjectList/index.tsx

```diff
@@ -26,6 +27,8 @@ const ProjectList: React.FC<Props> = ({ workspaceId }) => {
     hasMoreProjects,
     modalShown,
     openModal,
+    projectType,
+    prjTypeSlctOpen,
```

Comment:
FYI please be careful what words you shorten and make sure they are appropriate shortenings. For example, developer -> dev is normal, and dev is even colloquial. But Select -> slct is very much confusing and not easy to read.

Reply:
modified